### ISSUE TITLE
setup_suite and teardown_suite still run when a suite is disabled

### ISSUE THREAD

**Issue by chriswatrous** [lemoncheesecake/lemoncheesecake#16]
Cool test framework. I really like the html report. 

Minor problem: the setup and teardown still run when a suite is disabled. The scenario is I have a test suite that is working locally but the setup part doesn't run in our CI environment yet so I need to disable the suite.

### Versions
MacOS 10.15.3
Python 3.7.7
lemoncheesecake  1.4.7

### To Reproduce
```
lcc bootstrap somedir
```

create file suites/demo.py:
```
import lemoncheesecake.api as lcc

@lcc.suite('Demo')
@lcc.disabled()
class demo:
    def setup_suite(self):
        raise RuntimeError('🤮')

    def teardown_suite(self):
        raise RuntimeError('💥')

    @lcc.test('something')
    def something(self):
        pass
```

run with
```
lcc run --exit-error-on-failure; echo $?
```

### Expected Behavior

Setup and teardown should not run if the entire suite is disabled.

### Actual Behavior
```
==================================== demo =====================================
 --  1 # demo.something

Statistics :
 * Duration: 0.004s
 * Tests: 1
 * Successes: 0 (0%)
 * Failures: 0
 * Disabled: 1

1
```

The failure shows up in the html report.

For comparison, if I disable a test suite in mocha (the describe block) the before hook doesn't run. It does run if I have the describe block enabled but every test case individually disabled.
```
describe.skip('', () => {
    before(() => {
        console.log('in before');
    });

    it('', () => {
        console.log('in test case');
    });
});
```

**Comment by ndelon** [comment]
Hello Chris,

Thanks for the kind remark.

About the issue itself: this is clearly a bug and not the expected behavior. I think I will be able to fix it in the next few days.

Waiting for this, here is a possible workaround for your CI pipeline: you may pass the `--enabled` argument to `lcc run`, in that case the disabled tests will be filtered out and the setup/teardown methods won't be executed (but the disabled tests won't appear at all in the report).

Regards.

**Comment by ndelon** [comment]
Hello Chris,

The issue has been fixed in the latest 1.4.8 release.

Regards.

**Comment by chriswatrous** [comment]
It's working for me now. Thanks again.
